src/main/common/components/payment-intention/payment-date.njk

```diff
@@ -14,6 +14,10 @@
           </div>
         {% endif %}
 
+        {% if featureToggles.testingSupport and disposableIncome !== undefined %}
+          <p>Disposable Income value: {{ disposableIncome }}</p>
+        {% endif %}
```

Comment:
Could it be worth adding a style that makes it clear this is not normal content? Yellow background/border or something?

Reply:
have updated the content to make it clearer